generatebundlefile/hack/pullpush.sh

```diff
@@ -24,24 +24,15 @@
 
 set -e
 
-aws ecr-public get-login-password --region us-east-1 | HELM_EXPERIMENTAL_OCI=1 helm registry login --username AWS --password-stdin public.ecr.aws
-
 export local_ecr_public="public.ecr.aws/f5b7k4z5"
```

Comment:
This should really be done in your `~/.bash_profile` or the like, and then you can check for the value here and error if it isn't defined:
```sh
ECR_PUBLIC="${ECR_PUBLIC:?no public registry specified}"
```

Reply:
This is used by the CI, and figured it made more sense to put in MAKE than in bash scripts.